README.md

```
@@ -2026,23 +2026,28 @@ ctest --test-dir build-debug
 
 ## License
 
-Aleph-w is licensed under the **GNU General Public License v3.0**.
+Aleph-w is licensed under the **MIT License**.
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)